scprojects/views.py

```diff
@@ -85,18 +100,31 @@ def add_project(request):
             contributors=contributors_list,
         )
         new_project.save()
-        return JsonResponse({"status": "success"})
+        return JsonResponse({"result": "success"})
     else:
-        return JsonResponse({"status": "error", "reason": "authentication required"})
+        return JsonResponse({"result": "error", "reason": "authentication required"})
 
 
+@csrf_exempt
+@authentication_classes([TokenAuthentication])
+@permission_classes([IsAuthenticated])
 def login(request):
     token = ''
     if request.user.is_authenticated:
-        token = Token.objects.get(user=request.user)
-    return HttpResponseRedirect("http://localhost:3000/token/"+str(token))
+        try:
+            token = Token.objects.get(user=request.user)
+            print("Login token from DB ", token)
+            return HttpResponseRedirect("http://localhost:3000/token/"+str(token))
+        except Token.DoesNotExist:
+            print("Token.DoesNotExist")
```

Comment:
🗑 ?

Reply:
Keeping the print on L119 for debugging purposes, but removed the one on L116

Reply:
👍